src/main.js

```diff
@@ -725,6 +736,16 @@ class MiniGraphCard extends LitElement {
   }
 
   computeState(inState) {
+    if (this.config.state_map.length > 0) {
+      const stateMap = Number.isInteger(inState)
+        ? this.config.state_map[inState]
+        : this.config.state_map.find(state => state.value === inState);
+
+      if (stateMap) {
+        return stateMap.label;
+      }
```

Comment:
Wanted to console log/warn something to tell user that theirs configuration is not complete but eslint complains

Reply:
Yep, that's a good idea! You could add an exception for the line, like below:

https://github.com/kalkih/mini-graph-card/blob/34142fdefa5c1a4292c67acb3eda4ff1475fe9b6/src/main.js#L890-L891

Reply:
Ok will add it

Reply:
We could/should probably do a very simple log helper function in the future.